test/unit/materialized_view_historical_report_emission_data_test.sql

```diff
@@ -0,0 +1,94 @@
+set client_min_messages to warning;
+create extension if not exists pgtap;
+reset client_min_messages;
+
+begin;
+select * from no_plan();
+
+-- Test matview report exists in schema swrs_transform
+select has_materialized_view('swrs_transform', 'historical_report_emission_data', 'Materialized view historical_report_emission_data exists');
+
+-- Setup fixture
+insert into swrs_extract.eccc_xml_file (imported_at, xml_file) VALUES ('2018-09-29T11:55:39.423', $$
+<ReportData xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <ReportDetails>
+    <ReportID>800855555</ReportID>
+    <PrepopReportID></PrepopReportID>
+    <ReportType>R7</ReportType>
+    <FacilityId>666</FacilityId>
+    <OrganisationId>1337</OrganisationId>
+    <ReportingPeriodDuration>1999</ReportingPeriodDuration>
+    <ReportStatus>
+      <Status>In Progress</Status>
+      <Version>3</Version>
+      <LastModifiedBy>Donny Donaldson McDonaldface</LastModifiedBy>
+      <LastModifiedDate>2018-09-28T11:55:39.423</LastModifiedDate>
+    </ReportStatus>
+  </ReportDetails>
+  <ActivityData>
+    <ActivityPages>
+      <TotalEmissions>
```

Comment:
Looking at some sample reports `TotalEmissions` is sometimes empty, and when it is not the `TotalGroupType` values are not necessarily the same as what you have here. I don't think this is actually needed here either

Reply:
Nevermind this comment actually. If the total is 0 in the existing spreadsheet, then it will be 0 here as well.